### PR TITLE
Added Compose colour sample screen

### PR DESCRIPTION
**Pull request by jaanainkinen**
+ [ ] Check this if you have read and followed the [contributing guidelines](https://github.com/Skyscanner/backpack-android/blob/main/CONTRIBUTING.md)

Remember to include the following changes:
+ [ ] Component `README.md`
+ [ ] Tests
+ [ ] Docs (either update [backpack-docs](https://github.com/Skyscanner/backpack-docs) now, or create a follow up ticket)

If you are curious about how we review, please read through the [code review guidelines](https://github.com/Skyscanner/backpack/blob/main/CODE_REVIEW_GUIDELINES.md)

### CODE REVIEW

File: app/src/main/java/net/skyscanner/backpack/demo/compose/ColorsStory.kt [hunk 1]
```diff
@@ -0,0 +1,70 @@
+package net.skyscanner.backpack.demo.compose
```
Comment (bvitaliyg) [inline app/src/main/java/net/skyscanner/backpack/demo/compose/ColorsStory.kt:1]: Please add copyright header

File: app/src/main/java/net/skyscanner/backpack/demo/compose/ColorsStory.kt [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+      ColorSampleRow(token = item)
+    }
+  }
+}
+
+@Composable
+private fun ColorSampleRow(token: Token<Color>) {
+  Row(
+    modifier = Modifier
+      .height(56.dp),
```
Comment (bvitaliyg) [inline app/src/main/java/net/skyscanner/backpack/demo/compose/ColorsStory.kt:40]: No new line

File: app/src/main/java/net/skyscanner/backpack/demo/compose/ColorsStory.kt [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    verticalAlignment = Alignment.CenterVertically,
+  ) {
+    BpkText(
+      text = token.name,
+      modifier = Modifier
+        .weight(1f)
+        .padding(BpkSpacing.Base)
```
Comment (bvitaliyg) [inline app/src/main/java/net/skyscanner/backpack/demo/compose/ColorsStory.kt:47]: Please add a trailing comma

File: app/src/main/java/net/skyscanner/backpack/demo/compose/ColorsStory.kt [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+    )
+    Box(
+      modifier = Modifier
+        .height(56.dp)
+        .width(112.dp)
+        .padding(1.dp)
+        .background(token.value),
+      contentAlignment = Alignment.Center
```
Comment (bvitaliyg) [inline app/src/main/java/net/skyscanner/backpack/demo/compose/ColorsStory.kt:55]: Please add a trailing comma

File: app/src/main/java/net/skyscanner/backpack/demo/compose/ColorsStory.kt [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+    ) {
+      BpkText(
+        text = colorToHex(color = token.value),
+        color = if (token.value.luminance() > 0.5) Color.Black else Color.White
```
Comment (bvitaliyg) [inline app/src/main/java/net/skyscanner/backpack/demo/compose/ColorsStory.kt:59]: Please add a trailing comma

**Review by bvitaliyg — CHANGES_REQUESTED** [review]
Congrats on your first PR! Just a few minor comments :)

**Review by bvitaliyg — APPROVED** [review]
💯

**Review by marianeum — APPROVED** [review]
great first PR 🎉

**Pull request merged**